web/src/components/review-form.jsx

```diff
@@ -98,6 +103,24 @@ export default class ReviewForm extends React.Component {
     const offset = this.getOffset();
     const curSentences = this.props.sentences.slice(offset, offset + PAGE_SIZE);
 
+    if (mobileUI) {
+      return (
+        <Cards onEnd={() => this.onSubmit({'preventDefault': function () {}})} className="master-root">
```

Comment:
is the 'preventDefault' part here necessary?

Reply:
Yes, otherwise it errors, as `onSubmit` expects an object with the `preventDefault` method as its parameter. This whole thing is going to get changed to something else than onSubmit though.